node/src/chain_spec/phala.rs

```diff
@@ -234,8 +235,14 @@ fn genesis(
         vesting: phala_parachain_runtime::VestingConfig { vesting: vec![] },
         democracy: Default::default(),
         phragmen_election: Default::default(),
+        phala_registry: registry_override.unwrap_or(phala_parachain_runtime::PhalaRegistryConfig {
+            workers: Vec::new(),
+            gatekeepers: Vec::new(),
+            benchmark_duration: 50,
+        }),
+        phala_computation: Default::default(),
         polkadot_xcm: phala_parachain_runtime::PolkadotXcmConfig {
-            safe_xcm_version: Some(3),
+            safe_xcm_version: Some(2),
```

Comment:
@tolak Note that the version will not change in this upgrade. This line of the code only changes the genesis storage in a new chain.